src/backend/x64/a32_jitstate.cpp

```diff
@@ -201,4 +203,21 @@ void A32JitState::SetFpscr(u32 FPSCR) {
     }
 }
 
+constexpr u32 ASID_MASK = A32::LocationDescriptor::ASID_MASK;
+constexpr u32 ASID_BIT_SHIFT = A32::LocationDescriptor::ASID_BIT_SHIFT;
+constexpr u32 ASID_BIT_COUNT = 5;
+
+u8 A32JitState::Asid() const {
+    return (upper_location_descriptor >> ASID_BIT_SHIFT) & ASID_MASK;
```

Comment:
```suggestion
    return static_cast<u8>((upper_location_descriptor >> ASID_BIT_SHIFT) & ASID_MASK);
```

otherwise this is truncating a `u32` implicitly.